cfme/intelligence/reports/menus.py

```diff
@@ -171,7 +172,15 @@ def manage_folder(self, group, folder=None):
             view.save_button.click()
 
 
-@navigator.register(ReportMenu)
+@attr.s
+class ReportMenusCollection(BaseCollection):
+    """Collection object for the
+    :py:class:'cfme.intelligence.reports.ReportMenu'."""
+
+    ENTITY = ReportMenu
+
+
+@navigator.register(ReportMenusCollection)
```

Comment:
I think this navigation class should be registered against the entity.

Note that the step method references `self.obj.group`, which will raise an AttributeError against the collection - the collection doesn't have a `group` attribute, the entity does.